associations/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] This method supposes to add a new rental and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please fix that.
[Hint] It will be like that:

```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
end
```